docs/DEEP_TRACING.md

```diff
@@ -0,0 +1,11 @@
+## Design blueprint: Deep tracing
```

Comment:
Random file?

Reply:
+1  yeah looks like an accidental git add.

Reply:
Thanks for noticing, gents!